src/main/java/org/folio/bulkops/service/ErrorService.java

```diff
@@ -236,13 +236,22 @@ public String uploadErrorsToStorage(UUID bulkOperationId) {
       var errorsFileName = LocalDate.now() + operationRepository.findById(bulkOperationId)
         .map(BulkOperation::getLinkToTriggeringCsvFile)
         .map(FilenameUtils::getName)
-        .map(fileName -> "-Committing-changes-Errors-" + fileName)
+        .map(fileName -> ERROR_COMMITTING_FILE_NAME_PREFIX + fileName)
         .orElse("-Errors.csv");
       return remoteFileSystemClient.put(new ByteArrayInputStream(errorsString.getBytes()), bulkOperationId + "/" + errorsFileName);
     }
     return null;
   }
 
+  public String uploadErrorToStorage(UUID bulkOperationId, String fileNamePrefix, String errorString) {
+    var errorsFileName = LocalDate.now() + operationRepository.findById(bulkOperationId)
```

Comment:
Looks this part is similar  for  public String uploadErrorsToStorage(UUID bulkOperationId) and public String uploadErrorToStorage(UUID bulkOperationId, String fileNamePrefix, String errorString) {
. Can it be extracted as separate method?